deployment/tfhub/object-detection/base64/object-detection-base64-deploy.ipynb

```diff
@@ -0,0 +1,390 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Object Identification (TensorFlow Hub) - Base64 Images"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook identifies whether a given object is present in a set of images. If the object is found, it returns a dataframe with the file name, score and bounding boxes.\n",
+    "We are using a dataset from [UCF](https://www.crcv.ucf.edu/data/GMCP_Geolocalization/#Dataset) that contains 62,058 high quality Google Street View images, and [TensorFlow Hub](https://www.tensorflow.org/hub), a library and platform for transfer learning."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Dependencies"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook has been tested with Python 3.10.6 and the following package versions:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install dill==0.3.6\n",
+    "!pip install Pillow==9.3.0\n",
+    "!pip install tensorflow==2.11.0\n",
+    "!pip install tensorflow-hub==0.12.0\n",
+    "!pip install urllib3==1.26.13\n",
```

Comment:
I think we actually do not need `urllib3` in any of these notebooks. It looks like we're using the [standard library `urllib3.request`](https://docs.python.org/3/library/urllib.request.html).